esp8266-fastled-webserver/common.h

```diff
@@ -294,10 +292,11 @@ void drawSpiralAnalogClock89();
 void drawSpiralAnalogClock21and34();
 void drawSpiralAnalogClock13_21_and_34();
 void drawSpiralAnalogClock34_21_and_13();
-void antialiasPixelAR(uint8_t angle, uint8_t dAngle, uint8_t startRadius, uint8_t endRadius, CRGB color, CRGB leds[] = leds, int _NUM_PIXELS = NUM_PIXELS);
 #endif
 
 // noise.h -- always defined
+void paletteNoise();
```

Comment:
Here also.  One goal is to minimize the number of functions that are put into common headers.  Basically, not placing them here makes it easier to know that you can change those functions, because if they're not exposed via common.h, then those functions are local to that single compilation unit.

In fact, I'd recommend using an inline anonymous namespace in most of the source files, with only the exported functions existing outside the namespace.  This makes it impossible to have collisions with items declared within that namespace, which further helps with this.

When needed, place the local function prototypes at the top of the file / namespace where they are implemented.

It's just a small step towards segmenting the functionality of the code.

Reply:
Also in the patterns list: https://github.com/jasoncoon/esp8266-fastled-webserver/blob/a64f66aedfc824795f899ded4fc71e7a27d5333a/esp8266-fastled-webserver/esp8266-fastled-webserver.ino#L162